code/api/src/modules/userProducts/types.js

```diff
@@ -0,0 +1,20 @@
+// Imports
+import { GraphQLObjectType, GraphQLString, GraphQLInt, GraphQLBoolean } from 'graphql'
+import UserType from '../user/types'
+import ProductType from '../product/types'
+
+// Crate type
```

Comment:
Okay so in addition to using `Sequelize` to set up the tables in psql, we're also using `GraphQL` to make a new object type with `name` key being our table title/key and our `fields` method returning an object with typed property values. I assume that we do this to use cool graphql methods to manipulate the data/draw from multiple tables/get specific value fields?

Reply:
It's importing the relationship so we can call it in the graphql schema as the object - it's one of two ways for GraphQL to know that a relationship is in place.